### DISCUSSION TITLE
开启snapline后，document绑定的mouseup事件不会触发

### DISCUSSION THREAD

**Discussion by superchenwb** [antvis/X6#1732] [Q&A]
如题：开启snapline后，document绑定的mouseup事件不会触发，导致dragmove一直被触发

https://codesandbox.io/s/tender-browser-4fz3n

**Comment by NewByVector** [comment]
在 graph 的配置中增加 `preventDefaultMouseDown: false` 能解决你的问题。

**Reply by superchenwb** [reply]
测试过没有效果，看了代码，发现preventDefaultMouseDown无法阻止onMouseUp方法中的e.stopImmediatePropagation()，所以是无效的
![code](https://user-images.githubusercontent.com/17635660/172289964-b913bbef-e605-41fc-94b0-be0b29e7984e.png)

**Comment by kabudaLiu** [comment]
后续是怎么解决了的呢
